src/main/java/seedu/address/logic/commands/ListIncompleteCommand.java

```diff
@@ -0,0 +1,18 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.Model;
+
+/**
+ * Lists all incomplete tasks in the task book to the user.
+ */
+public class ListIncompleteCommand extends Command {
+
+    public static final String COMMAND_WORD = "listincomplete";
+    public static final String MESSAGE_SUCCESS = "Listed all incomplete tasks";
+
+    @Override
+    public CommandResult execute(Model model) {
+        model.updateFilteredTaskList(task -> !task.isCompleteProperty().get());
+        return new CommandResult(MESSAGE_SUCCESS);
+    }
+}
```

Comment:
Should we name the class ListIncompleteTasksCommand instead? I think it provides more clarity that way.

Reply:
I agree, ListIncompleteCommand could be ambiguous, we should put task inside the name to specify what it is doing.